pkgs/servers/etcd/3.5.nix

```diff
@@ -39,6 +39,8 @@ let
     ldflags = [ "-X go.etcd.io/etcd/api/v3/version.GitSHA=GitNotFound" ];
 
     meta = commonMeta;
+
+    passthru.tests = { inherit (nixosTests) etcd etcd-cluster; };
```

Comment:
Sorry, I should have specified that this needs to be in the main derivation so ofborg will test it automatically.

```diff
diff --git a/pkgs/servers/etcd/3.5.nix b/pkgs/servers/etcd/3.5.nix
index 643eafaa353..6420ca87f3a 100644
--- a/pkgs/servers/etcd/3.5.nix
+++ b/pkgs/servers/etcd/3.5.nix
@@ -39,8 +39,6 @@ let
     ldflags = [ "-X go.etcd.io/etcd/api/v3/version.GitSHA=GitNotFound" ];
 
     meta = commonMeta;
-
-    passthru.tests = { inherit (nixosTests) etcd etcd-cluster; };
   };
 
   etcdutl = buildGoModule rec {
@@ -76,6 +74,8 @@ symlinkJoin {
   version = etcdVersion;
   meta = commonMeta;
 
+  passthru.tests = { inherit (nixosTests) etcd etcd-cluster; };
+
   paths = [
     etcdserver
     etcdutl
```

Reply:
No problem! I am working on tests itself now :)